src/core/utils/testing/fake-objects.ts

```diff
@@ -232,7 +232,8 @@ export const fakeInternalConfig = fakeObjectCreator<InternalConfig>(() => {
     },
     zip: {
       artifactTemplate: '{{name}}-{{version}}.zip',
-      ignoredSources: [],
+      includedSources: [],
+      excludedSources: [],
```

Comment:
If we're making a breaking change, can we change this to `includeSources` and `excludeSources`? I think most config fields don't have the `d` at the end. Here's vitest:

<img width="201" alt="Screenshot 2024-01-29 at 10 42 49 AM" src="https://github.com/wxt-dev/wxt/assets/10101283/0ddbc319-e0f9-482a-9656-fe9dddff2bbd">

Reply:
Oh yes, definitely. I wanted to do this, but given it was `ignored` (with a `d`), I thought you may have preferred it that way. I'm on it.